src/styles/components/_menu.scss

```diff
@@ -1,10 +1,31 @@
-$interface-types: (
```

Comment:
We will want to use palette, and we will want to define a role color for the menu. @wwqrd can you explain how to do this since I think you set it up?